src/main/java/ru/lanwen/wiremock/ext/WiremockResolver.java

```
@@ -40,15 +36,13 @@ public WiremockResolver() {
     }
 
     @Override
-    public void afterEach(ExtensionContext testExtensionContext) throws Exception {
-        if (server == null || !server.isRunning()) {
```

fail fast is preferrable